### ISSUE TITLE
Grid column resizing does not work in Chrome when page is zoomed

### ISSUE THREAD

**Issue by VicTachev** [telerik/kendo-ui-core#3623]
### Reproduction of the problem
1. Open the Column resizing demo -[http://demos.telerik.com/kendo-ui/grid/column-resizing](http://demos.telerik.com/kendo-ui/grid/column-resizing)
2.  Zoom the page to anything different than 100%
3. Try to resize a column

### Current behavior
The resize handler is not displayed and columns cannot be resized.

### Expected/desired behavior
Resize handler should be displayed.

### Environment

* **Kendo UI version:** 2017.3.913
* **jQuery version:** 1.12
* **Browser:** [Chrome 61]

**Comment by simonssspirit** [comment]
reported twice in forum ticket: 1127003

**Comment by theOrlin** [comment]
Also reported in tickets  1132278 and  1132395

**Comment by Stamo-Gochev** [comment]
Duplicate item: #3609

**Comment by Alex-Bubblemaster** [comment]
Also reported in Ticket ID: 113292. Raising priority.

**Comment by VicTachev** [comment]
Workaround available in the KB below:

[http://docs.telerik.com/kendo-ui/knowledge-base/grid-column-resize-not-working-in-chrome](http://docs.telerik.com/kendo-ui/knowledge-base/grid-column-resize-not-working-in-chrome)

**Comment by emilyytlin** [comment]
Will this be included in a future release, as opposed to a workaround?

**Comment by Stamo-Gochev** [comment]
@emilyytlin Yes, the fix will be included in our next release - either a latest internal build or a service pack.

**Comment by mehdymohajery** [comment]
in our case it only shows resize handler on 80% zoom!
resizing columns doesn't work even before zoom
you can see a recorded video of kendo online sample here:

 [column-resizing-issue](https://drive.google.com/file/d/0BxTBkkbV6obtSVVLQzMzVTNTQ2M/view?usp=sharing)

the video is recorded from [kendo demo fro column resizing](http://demos.telerik.com/aspnet-mvc/grid/column-resizing) with chrome 61.0.3163.100 (Official Build) (64-bit) on win10 x64

**Comment by VicTachev** [comment]
@mehdymohajery The issue can be seen also when scaling is enabled in the OS. Nevertheless, the [workaround from a previous commend](https://github.com/telerik/kendo-ui-core/issues/3623#issuecomment-334480021) should work when scaling is enabled.

With that said, the issue has been fixed and the fix will be available in the upcoming Service Pack.

**Comment by adrobyazko-softheme** [comment]
@VicTachev, @simonssspirit, @theOrlin ,@Stamo-Gochev, @Alex-Bubblemaster after applying mentioned "Solution" we have more serious issue.
From time to time resizing fails even for normal (100%) size.
Could you please have a look at the logs?
If you need more info - please let me know.

![image](https://user-images.githubusercontent.com/3669463/33772474-6c97d9ce-dc3d-11e7-92a6-7ecb58b35779.png)
[KendoGridResizing_SolutionErros.log](https://github.com/telerik/kendo-ui-core/files/1543058/KendoGridResizing_SolutionErros.log)

**Comment by Stamo-Gochev** [comment]
@adrobyazko-softheme
Can you confirm that you are actually testing with a Kendo UI release version (2017.3.1026 is currently the latest) and not with the workaround? In addition, can you send a Dojo example with your configuration and a video, which demonstrates the actions that are necessary in order to reproduce the problem?

**Comment by adrobyazko-softheme** [comment]
Hello @Stamo-Gochev we have  2017.3.913.545 version (not latest).
Could be issue related with not latest version?

![image](https://user-images.githubusercontent.com/3669463/33821711-8f58d554-de5d-11e7-9c92-37af7a444333.png)

**Comment by Stamo-Gochev** [comment]
The fix is released in R3 2017 SP1 (version 2017.3.1018), so it is not included in 2017.3.913.

Can you test with version 2017.3.1018 (or later) and send an example and a video if the problem still occurs?

**Comment by adrobyazko-softheme** [comment]
Hi @Stamo-Gochev,
Unfortunatelly I can't update to latest version. It seems that smt. wrong with nuget.telerik.com. 

I can login with my creds to https://www.telerik.com/account/ and https://nuget.telerik.com/nuget/ but VS 2015 returns errors when I'm trying update nuget package.

Could you check this issue?

![image](https://user-images.githubusercontent.com/3669463/33877195-523ff94e-df30-11e7-8013-bf1d834f8e8a.png)

**Comment by Stamo-Gochev** [comment]
@adrobyazko-softheme 
The Nuget feed should target https://api.nuget.org/v3/index.json, but it looks like it is currently targeting v2.

Can you send us a support ticket with a detailed information about the problem so that the support team can provide additional assistance? This also applies to any further questions regarding the the column resizing fix, so we can keep the current issue concise and dedicated to a single topic.

**Comment by adrobyazko-softheme** [comment]
Hi @Stamo-Gochev I've updated Kendo to latest version 2017.3.1206.
It seems issue solved.
Could you confirm that "fix" was included to the release?

Thanks.

**Comment by nanbs** [comment]
Hi all, I am having the issue with resizing the kendo grid when page is zoomed in Chrome. I can only resize at 100%. I have updated Chrome to the latest version (Version 63.0.3239.132) and we are using Progress® Kendo UI® Version: 2017.3.1026 (no new version available on my account) and Progress® Testing Framework Version: 2017.2.824.1 (new version available: 2017.3.1206.1). In the post above, it was mentioned that Kendo was updated to latest version 2017.3.1206 but I don't see this version in my TELERIK account. Will this help with this issue and where can I download it from? Thanks.
